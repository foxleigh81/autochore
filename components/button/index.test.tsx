import React from 'react';
import { cleanup, render, RenderResult } from '@testing-library/react';

import Button from './index';

let renderResult: RenderResult;

describe('Button component', () => {
  beforeEach(() => {
    renderResult = render(<Button onClick={() => true}>Test</Button>);
  });

  afterEach(() => {
    cleanup();
  });

  it('should match the most recent snapshot', () => {
    expect(renderResult.baseElement).toMatchSnapshot();
  });
});
