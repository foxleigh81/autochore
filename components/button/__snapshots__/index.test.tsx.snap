// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should match the most recent snapshot 1`] = `
<body>
  <div>
    <button
      class="button button-primary"
      tabindex="0"
    >
      Test
    </button>
  </div>
</body>
`;
